spec.md

```diff
@@ -2,6 +2,15 @@
 
 The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
 
+# Abstract
+
+The OCI Runtime Specification aims to specify the inputs and outputs for running containers on various platforms.
+
+A container's input or configuration is specified as the `config.json` for the supported platforms and details fields that enable containers to run on that platform.
+It contains host specific information for the creation of a container.
+
+The outputs of a container are defined as both actions that can be performed on a container after it has been created and the environment details that the container provides to applications running inside of it.
```

Comment:
These don't sound like “the outputs of a container” to me.  Actions are more like additional inputs.  The container environment is more of a runtime output, and that's probably what you're trying to focus on here.  I'd consider container output to be things like filesystem alterations which persist after container exit (e.g. simulation results written to a bind-mounted volume).  And containers which host services don't fit cleanly into the the idea of “container output” at all.  Maybe de-emphasize input/output and instead sketch the configuration, actions, and lifecycle in a few sentences each?  Something like:

> Containers are [configured](config.md) with metadata necessary to implement standard operations against the container. This includes the process to run, environment variables to inject, sandboxing features to use, etc.  Runtimes [create](runtime.md#create) containers from that configuration, and several [operations](runtime.md#operations) are defined to manage the container over its [lifecycle](runtime.md#lifecycle).